apps/codeforafrica/package.json

```diff
@@ -48,6 +51,7 @@
     "@types/react": "^17.0.44",
     "babel-jest": "^27.5.1",
     "config": "workspace:*",
+    "@commons-ui/core": "workspace:*",
```

Comment:
why do we have this both in dependencies and dev dependencies

Reply:
@KhadijaMahanga Yeah, that should be in dependancies alone. Removed it.